x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_stops.js

```diff
@@ -43,9 +43,20 @@ export function getFirstUnusedSymbol(iconStops) {
   return firstUnusedSymbol ? firstUnusedSymbol.value : DEFAULT_ICON;
 }
 
-export function IconStops({ field, getValueSuggestions, iconStops, onChange }) {
-  return iconStops.map(({ stop, icon }, index) => {
-    const onIconSelect = (selectedIconId) => {
+export function IconStops({
+  field,
+  getValueSuggestions,
+  iconStops,
+  onChange,
+  onCustomIconsChange,
+  customIcons,
+}) {
+  return iconStops.map(({ stop, icon, iconSource }, index) => {
+    const { svg, label } =
```

Comment:
customIcons.find(({ symbolId }) => symbolId === icon) and getMakiSymbol(icon) can return undefined and you can not spread undefined. Add logic to handle case where icon is not found